resources/assets/components/pages/VoterRegistrationDrivePage/Beta/HeroSection.js

```diff
@@ -15,9 +17,41 @@ const HeroSection = ({ user, campaignInfo, modalToggle }) => {
   } = campaignInfo;
 
   /**
-   * TODO: Check for a voting-reasons query parameter, and render values in quote if present.
-   * @see https://www.pivotaltracker.com/story/show/172087475
+   * Query url for voting-reasons param.
+   *
+   * @return {String}
    */
+
+  const formatQuote = () => {
+    const votingReasonsQuery = query('voting-reasons');
+
+    if (!votingReasonsQuery) {
+      return null;
+    }
+
+    const votingReasonsValues = votingReasonsQuery.split(',');
+
+    if (votingReasonsValues.length === 1) {
+      return ` like ${votingReasons[votingReasonsValues[0]]}`;
+    }
+
+    if (votingReasonsValues.length > 1) {
+      let result = '';
+
+      votingReasonsValues.forEach((value, index) => {
+        if (index === votingReasonsValues.length - 1) {
+          result = `${result} and ${votingReasons[value]}`;
```

Comment:
This is missing its trailing comma, which we could easily fix with:
```suggestion
          result = `${result}, and ${votingReasons[value]}`;
```